pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -69,7 +69,7 @@ func getDefaultCircuitBreakerThresholds() *cluster.CircuitBreakers_Thresholds {
 // For outbound: Cluster for each service/subset hostname or cidr with SNI set to service hostname
 // Cluster type based on resolution
 // For inbound (sidecar only): Cluster for each inbound endpoint port and for each service port
-func (configgen *ConfigGeneratorImpl) BuildClusters(proxy *model.Proxy, push *model.PushContext) []*cluster.Cluster {
+func (configgen *ConfigGeneratorImpl) BuildClusters(proxy *model.Proxy, push *model.PushContext, w *model.WatchedResource) []*cluster.Cluster {
```

Comment:
I believe it is better to not pollute the BuildClusters implement, for grpc gen, can you call buildOutboundClusters if possible.

Reply:
I think I agree as I test this further.